frontend/src/entry/overview/EntryRoleList.tsx

```diff
@@ -1,23 +1,61 @@
-import { Grid } from '@mui/material'
-import EntryCardEntityRoleRow from 'src/entry/overview/EntryCardEntityRoleRow'
-import { EntryInterface } from 'types/types'
+import GroupIcon from '@mui/icons-material/Groups'
+import UserIcon from '@mui/icons-material/Person'
+import { Grid, Stack } from '@mui/material'
+import { Fragment, useMemo } from 'react'
+import UserDisplay from 'src/common/UserDisplay'
+import EntryCardRolesChipSet from 'src/entry/overview/EntryCardRolesChipSet'
+import { CollaboratorEntry, EntryInterface } from 'types/types'
 
 type EntryCardRoleDialogProps = {
   entry: EntryInterface
 }
 
 export default function EntryRoleList({ entry }: EntryCardRoleDialogProps) {
+  const rows = useMemo(
+    () =>
+      entry.collaborators.map((collaborator) => (
+        <Fragment key={collaborator.entity}>
+          <Grid item xs={6}>
+            <Stack direction='row' alignItems='center' spacing={1}>
+              <EntryCollaboratorIcon entryCollaborator={collaborator} />
```

Comment:
Sorry my earlier comments weren't clear. I didn't change the following in the code snippet I provided, even though I'd asked for it in another comment.

You should be using the new `EntityIcon` that you moved into `frontend/src/entry/EntityIcon.tsx`
```
// At the top of this file
import EntityIcon from 'src/entry/EntityIcon`

// Then replace this line with the following
<EntityIcon entryCollaborator={collaborator} />
```

Reply:
Sorry, yeah just forgot to change it. You were clear, dw. Changed.